app/src/main/res/layout/wavegen_intent_dialog.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@color/colorPrimary"
+        android:gravity="center"
+        android:padding="@dimen/padding_title_dialog_wave"
+        android:text="@string/open_instrument"
+        android:textColor="@color/white"
+        android:textSize="@dimen/dialog_text_size" />
+
+    <Space
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/dialog_space_width" />
+
+    <in.goodiebag.carouselpicker.CarouselPicker
+        android:id="@+id/carousel"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content" />
+
+    <Space
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/dialog_space_width" />
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <Button
+            android:id="@+id/none_btn"
```

Comment:
Why is it called the none button? Doesn't makes any sense when I press None when I am in the dialog. What will the user expect? It should be renamed as `Cancel`
See: https://developer.android.com/guide/topics/ui/dialogs